src/components/Login.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useAuth } from '../contexts/AuthContext';
+
+export function Login() {
+  const { signInWithGoogle } = useAuth();
+
+  return (
+    <div className="login-container">
+      <h1>Welcome to Kanban Board</h1>
+      <p>Please sign in to add or move tasks</p>
+      <button onClick={signInWithGoogle} className="login-button">
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
  <button onClick={async () => {
    try {
      await signInWithGoogle();
    } catch (error) {
      console.error('Authentication failed:', error);
      // Handle error appropriately (e.g., show error message to user)
    }
  }} className="login-button">
```
Add error handling for the Google sign-in process to gracefully handle authentication failures and provide user feedback



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;